e2e/test/scenarios/embedding/embedding-snippets.cy.spec.js

```diff
@@ -8,6 +8,10 @@ import { IFRAME_CODE, getEmbeddingJsCode } from "./shared/embedding-snippets";
 
 const features = ["none", "all"];
 
+function codeBlock() {
```

Comment:
> Play around with the parameter settings, the parameters should be highlighted

They are not highlighted after navigating away and back.


https://github.com/user-attachments/assets/0a998249-0564-49c3-9750-ead6e7aa84bd

Reply:
Confirmed to be fixed in the latest changes, thanks for adding the tests!

https://github.com/user-attachments/assets/b461770d-32c6-46fe-933d-0ce369ac5bb3